.travis.yml

```diff
@@ -26,9 +26,9 @@ env:
 
 matrix:
   include:
-    - php: '7.1'
-    - php: '7.2'
-    - php: '7.3'
+    - php: '7.1.29'
```

Comment:
Why do we require specific patch versions? IMHO we should always build using last patch version.

Reply:
Travis had issues with curl on its minor php versions. I tried to manually download a root certificate and to set a php option to point to this certificate but there was no changes.
Secifying the latest patch fixed the issue (see commit https://github.com/KnpLabs/Gaufrette/pull/533/commits/9224c747769dd945e1e5a5eb2dbd67c942383b20 ).

I can try to always use the latest version instead of hardcoding it.